patterns/design-patterns/src/main/java/com/baeldung/currying/Letter.java

```diff
@@ -0,0 +1,109 @@
+package com.baeldung.currying;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+
+public class Letter {
+    private String returningAddress;
+    private String insideAddress;
+    private LocalDate dateOfLetter;
+    private String salutation;
+    private String body;
+    private String closing;
+
+    Letter(String returningAddress, String insideAddress, LocalDate dateOfLetter, String salutation, String body,
+            String closing) {
+        this.returningAddress = returningAddress;
+        this.insideAddress = insideAddress;
+        this.dateOfLetter = dateOfLetter;
+        this.salutation = salutation;
+        this.body = body;
+        this.closing = closing;
+    }
+
+    Letter(String salutation, String body) {
+        this(null, null, LocalDate.now(), salutation, body, null);
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o)
+            return true;
+        if (o == null || getClass() != o.getClass())
+            return false;
```

Comment:
These should have `{}` around the body of the if - to be in line with most convention.

Reply:
:check

Reply:
:heavy_check_mark: